src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/subjectAccessRequest/SarAppointmentTimeType.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.subjectAccessRequest
+
+import com.fasterxml.jackson.annotation.JsonValue
+
+enum class SarAppointmentTimeType(@JsonValue val description: String) {
+  IMMEDIATE_UPON_RELEASE("Immediate upon release"),
+  NEXT_WORKING_DAY_2PM("Next working day 2pm"),
```

Comment:
worth saying either `before 2pm` or `after 2pm` in the string (can't remember which way round it is)